services/frontend-service/src/ui/App/index.tsx

```diff
@@ -73,6 +73,10 @@ export const App: React.FC = () => {
             .then(
                 (result: GetFrontendConfigResponse) => {
                     UpdateFrontendConfig.set({ configs: result, configReady: true });
+                    if (result.authConfig?.dexAuth?.enabled) {
+                        // eslint-disable-next-line no-console
+                        console.log('Dex is enabled lets redirect ');
```

Comment:
Won't this flood console.log?

Reply:
Yes, the next step is to remove this log and actually make the redirect. I can remove it